docs/enhancements/install-sno-using-ibi.md

```diff
@@ -0,0 +1,133 @@
+---
+title: install-sno-using-ibi
+authors:
+  - "@adriengentil"
+creation-date: 2024-03-28
+last-updated: 2024-03-28
+---
+
+# Support Image-Based Installation in Assisted-Installer to install SNO
+
+## Summary
+
+Currently, installing an SNO cluster using the Assisted-Installer takes up to
+~40min using the current flow which installs OCP from the ground-up.
+
+We can reduce the installation time to ~10min by leveraging the Image-Based
+installation method, and as a way to increase the success rate of SNO
+installations as we restore previously working setup.
```

Comment:
Maybe we should get more exact timings for this.

Does that 40 minutes include all the pre-install validations we're running or is that from the time the user hits the install button? We're going to still be running all those validations during discovery right?

Reply:
I agree. I believe @eranco74 already has some numbers in a doc somewhere

Reply:
The actual SNO installation takes 30 minutes.